css/properties/inset-area.json

```diff
@@ -0,0 +1,40 @@
+{
+  "css": {
+    "properties": {
+      "inset-area": {
```

Comment:
This one is missing a lot of values as sub features. Running the collector I got this: https://gist.github.com/Elchi3/7651d3f1bcfbe328645165866120010a. Do you want to add it?

Reply:
Sure, added.

One thing to flag is that these are not exclusively single, standalone values. They can be used on their own, but they can also be used in combination, as pairs of values. Is that an issue? I will of course be explaining all of this in the reference page.

Reply:
That's okay, if there are compat issues with the values standalone or in combination, we will see how to add it in the future. For now it is important to capture which values are part of the initial implementation.

It would be nice if you add spec_urls (this is our insurance that the values are valid per spec and not outdated)

Reply:
spec urls added!

I'll add spec urls for the other property subfeatures I added too.